src/app/modules/evaluate/component/evaluate-search-table/evaluate-search-table.component.html

```diff
@@ -7,26 +7,24 @@
       <th class="age">{{ 'evaluate.age' | translate }}</th>
     </tr>
   </table>
-  <cdk-virtual-scroll-viewport
-    class="viewport"
-    [style.height.px]="rows.length > 20 ? 20 * 20 : (rows.length + 1) * 20"
-    itemSize="19"
-    minBufferPx="200"
-    maxBufferPx="400"
-  >
+  <cdk-virtual-scroll-viewport class="viewport"
+                               [style.height.px]="rows.length > 20 ? 20 * 20 : (rows.length + 1) * 20"
+                               itemSize="19"
+                               minBufferPx="200"
+                               maxBufferPx="400">
     <table>
-      <tr
-        *cdkVirtualFor="let row of rows; templateCacheSize: 0"
-        class="clickable"
-        [class.fixer]="row.count >= 3"
-        (click)="onRowClick($event, row)"
-      >
+      <tr *cdkVirtualFor="let row of rows; templateCacheSize: 0"
+          class="clickable"
+          [class.fixer]="row.count >= 3"
+          (click)="onRowClick($event, row)">
         <td class="amount" *ngIf="!original">
           <span class="value">{{ row.amount }}</span>
         </td>
         <td class="amount" *ngIf="original">
-          <app-currency-frame [amount]="row.originalAmount" [currency]="row.originalCurrency">
+          <app-currency-frame [amount]="row.originalAmount" [currency]="row.originalCurrency" *ngIf="!showRatio(row)">
```

Comment:
you can *ngIf=""expressions; else template" to save having to run the `showRatio` check twice from opposite sides

Reply:
Changed in 48aec46